integr8ly/keycloak-operator
# Update dashboard

## Motivation

Keycloak could be deployed to multiple namespaces but currently the dashboard will only work for one namespace

## How

Add a dashboard variable called "namespace" to allow users to choose the namespace. Also update the queries to use this variable.

## Verification Steps

See https://grafana-route-middleware-monitoring.apps.integreatly-222b.openshiftworkshop.com/d/WkoLd4gZz/keycloak-copy?refresh=5s&orgId=1&var-namespace=sso

## Checklist:
